src/chainstate/stacks/index/marf.rs

```diff
@@ -79,10 +79,18 @@ use chainstate::stacks::index::Error as Error;
 
 use util::log;
 
+pub const BLOCK_HASH_TO_HEIGHT_MAPPING_KEY: &str = "__MARF_BLOCK_HASH_TO_HEIGHT";
+pub const BLOCK_HEIGHT_TO_HASH_MAPPING_KEY: &str = "__MARF_BLOCK_HEIGHT_TO_HASH";
```

Comment:
I think this is okay as-is, but is there code anywhere that ensures that these two values _cannot_ be written by `insert_batch()` or `insert()`?  i.e. we're sure that there's no way a Clarity smart contract can insert into the MARF using either of these keys?

Reply:
The Clarity MARF wrapper uses a different prefix, so there's no way that they'd have the same key as is. I'm not sure if there's a more stringent way to enforce this, though.